### PR TITLE
SSB: Major update

### PR DESCRIPTION
**Pull request by ghost**
Closes #68 as i added @Volco 's code from that PR (because conflicts)

### CODE REVIEW

**Comment by ghost** [comment]
Tested and works

File: mods/seasonal/moves.js [hunk 1]
```diff
@@ -33,6 +33,53 @@ let BattleMovedex = {
...
+		onPrepareHit: function (target, source) {
+			this.attrLastMove('[still]');
+			this.add('-anim', source, 'Howl', source);
+			this.add('-anim', source, 'Boomburst', source);
+		},
+		onTry: function (source) {
+			this.boost({atk: 2}, source, source, 'move: Noble Howl');
+			if (!(['', 'slp', 'frz'].includes(source.status))) {
+				source.cureStatus();
+			}
```
Comment (HoeenCoder) [inline mods/seasonal/moves.js]: These should be happening when the move hits

File: mods/seasonal/moves.js [hunk 2]
```diff
@@ -297,6 +386,32 @@ let BattleMovedex = {
...
+			this.attrLastMove('[still]');
+			this.add('-anim', source, "Entrainment", target);
+			this.add('-anim', source, "Lock On", target);
+		},
+		onHit: function (target) {
+			target.addVolatile('taunt');
+			target.addVolatile('embargo');
+			target.addVolatile('torment');
+			target.addVolatile('confusion');
+			target.addVolatile('healblock');
```
Comment (HoeenCoder) [inline mods/seasonal/moves.js]: `target, source` and pass source to .addVolatile

File: mods/seasonal/moves.js [hunk 3]
```diff
@@ -945,6 +1081,34 @@ let BattleMovedex = {
...
 		zMoveEffect: 'clearnegativeboosts',
 	},
+	// MajorBowman
+	blazeofglory: {
+		accuracy: true,
+		basePower: 0,
+		damageCallback: function (pokemon) {
+			let damage = pokemon.hp;
+			pokemon.faint();
+			return damage;
```
Comment (HoeenCoder) [inline mods/seasonal/moves.js:1089]: Did you test this? Not sure if the user faints mid move if it fails.

File: mods/seasonal/moves.js [hunk 4]
```diff
@@ -1609,6 +1773,28 @@ let BattleMovedex = {
...
+		pp: 10,
+		priority: 0,
+		flags: {},
+		onPrepareHit: function () {
+			this.attrLastMove('[still]');
+		},
+		onHit: function (target, source) {
+			this.useMove("healorder", source);
+			this.useMove("defendorder", source);
+			this.useMove("attackorder", source);
```
Comment (HoeenCoder) [inline mods/seasonal/moves.js]: You will want to pass source twice (as the target and source)

File: mods/seasonal/moves.js [hunk 5]
```diff
@@ -1609,6 +1773,28 @@ let BattleMovedex = {
...
+	// urkerab
+	holyorders: {
+		accuracy: true,
+		basePower: 0,
+		category: "Status",
+		id: "holyorders",
+		name: "Holy Orders",
+		pp: 10,
+		priority: 0,
+		flags: {},
```
Comment (HoeenCoder) [inline mods/seasonal/moves.js]: snatch: 1, is good for move like this.

File: mods/seasonal/moves.js (hunk 3 above)
Comment (ghost) [reply mods/seasonal/moves.js:1089]: I did, it works. The code was copied from final gambit

File: mods/seasonal/moves.js (hunk 5 above)
Comment (ghost) [reply mods/seasonal/moves.js]: ok

File: mods/seasonal/moves.js (hunk 4 above)
Comment (ghost) [reply mods/seasonal/moves.js]: for what reason?

Comment (Lycanium-Z) [reply mods/seasonal/moves.js]: for what reason?

Comment (HoeenCoder) [reply mods/seasonal/moves.js]: To prevent any issues for not having a source + clean code.

**Pull request merged**